ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -263,7 +263,8 @@ private[state] object Conversions {
       TransactionCoder.encodeTransaction(TransactionCoder.NidEncoder, ValueCoder.CidEncoder, tx),
```

Comment:
I think it will be cleaner if `encodeTransaction` were symetrical to decodeTransaction, meaning its output type should be `Raw.Transaction`.

Reply:
Good idea, done, thanks!